### ISSUE TITLE
Radar雷达图，v4.1.0.rc2版本axisLabel使用后崩溃，3.8.5版本没有问题

### ISSUE THREAD

**Issue by kl12000** [apache/incubator-echarts#8663]
以官网链接为例：http://echarts.baidu.com/examples/editor.html?c=radar

### One-line summary [问题简述]
Radar雷达使用axisLabel后，直接崩溃。
axisLabel:{
            show:true,
        }

echarts.js:38134 Uncaught TypeError: axisModel.getCategories is not a function
    at buildAxisLabel (echarts.js:38134)
    at AxisBuilder.axisTickLabel (echarts.js:37751)
    at AxisBuilder.add (echarts.js:37626)
    at Array.forEach (<anonymous>)
    at each$1 (echarts.js:524)
    at ExtendedClass.<anonymous> (echarts.js:42087)
    at Array.forEach (<anonymous>)
    at each$1 (echarts.js:524)
    at ExtendedClass._buildAxes (echarts.js:42086)
    at ExtendedClass.render (echarts.js:42068)
﻿

38133    var labelLayout = innerTextLayout(opt.rotation, labelRotation, opt.labelDirection);
38134    var rawCategoryData = axisModel.getCategories(true);

### Version & Environment [版本及环境]
+ ECharts version [ECharts 版本]:v4.1.0.rc2 
+ Browser version [浏览器类型和版本]:chorme，IE
+ OS Version [操作系统类型和版本]:win7

### ECharts option [ECharts配置项]

```javascript
option = {
    title: {
        text: '基础雷达图'
    },
    tooltip: {},
    legend: {
        data: ['预算分配（Allocated Budget）', '实际开销（Actual Spending）']
    },
    radar: {
        // shape: 'circle',
        name: {
            textStyle: {
                color: '#fff',
                backgroundColor: '#999',
                borderRadius: 3,
                padding: [3, 5]
           }
        },
        indicator: [
           { name: '销售（sales）', max: 6500},
           { name: '管理（Administration）', max: 16000},
           { name: '信息技术（Information Techology）', max: 30000},
           { name: '客服（Customer Support）', max: 38000},
           { name: '研发（Development）', max: 52000},
           { name: '市场（Marketing）', max: 25000}
        ],
        axisLabel:{
            show:true,
        }
    },
    series: [{
        name: '预算 vs 开销（Budget vs spending）',
        type: 'radar',
        // areaStyle: {normal: {}},
        data : [
            {
                value : [4300, 10000, 28000, 35000, 50000, 19000],
                name : '预算分配（Allocated Budget）'
            },
             {
                value : [5000, 14000, 28000, 31000, 42000, 21000],
                name : '实际开销（Actual Spending）'
            }
        ]
    }]
};
```

**Comment by 100pah** [comment]
Fixed in #10036

**Comment by 100pah** [comment]
Fixed in #10036
